tests/foreman/cli/test_role.py

```diff
@@ -1265,3 +1273,124 @@ def test_negative_assign_org_admin_to_ldap_user_group(self):
 
         :CaseLevel: System
         """
+
+
+class SystemAdminTestCases(CLITestCase):
+
+    @upgrade
+    @tier3
+    def test_system_admin_role_end_to_end(self):
+        """Test System admin role with a end to end workflow
+
+        :id: da6b3549-d1cf-44fc-869f-08d15d407fa2
+
+        :steps:
+
+            1. Create a System admin role user1
+            2. Login with the user1 and change global settings
+                "Out of sync interval" to 31
+            3. Create user2 with system admin role
+            4. Login with user2 to create a Organization
+            5. Clone a Org-admin role
+            6. Edit the Architecture Filter and search name  =  x86_64
+            7. Create a User with Cloned Org admin
+            8. Login with user.
+
+        :expectedresults:
+
+            1. User should be assigned with System Admin role.
+            2. User with sys admin role should be able to update settings
+            3. User with sys admin role should be able to create users and
+                assign Organizations to them.
+            4. System Admin role should be able to create Organization admins
+            5. User with sys admin role should be able to edit filters on roles
+
+        :CaseLevel: System
+        """
+        org = make_org()
+        location = make_location()
+        common_pass = gen_string('alpha')
+        role = Role.info({'name': 'System admin'})
+        system_admin_1 = make_user({
+            'password': common_pass,
+            'organization-ids': org['id'],
+            'location-ids': location['id']
+            })
+        User.add_role({
+            'id': system_admin_1['id'],
+            'role-id': role['id']
+            })
+        Settings.with_user(
+            username=system_admin_1['login'],
+            password=common_pass).set({
+                'name': "outofsync_interval",
+                'value': "32"
+                })
+        sync_time = Settings.list({
+            'search': 'name=outofsync_interval'
+            })[0]
+        # Asserts if the setting was updated successfully
+        self.assertEqual('32', sync_time['value'])
```

Comment:
@san7ket Can you please add logic to revert the change as settings are global? There may be chances that it will affect other tests.